src/cleanup_queue.cc

```diff
@@ -8,27 +8,22 @@ namespace node {
 std::vector<CleanupQueue::CleanupHookCallback> CleanupQueue::GetOrdered()
     const {
   // Copy into a vector, since we can't sort an unordered_set in-place.
-  std::vector<CleanupHookCallback> callbacks(cleanup_hooks_.begin(),
-                                             cleanup_hooks_.end());
+  std::vector callbacks(cleanup_hooks_.begin(), cleanup_hooks_.end());
```

Comment:
Good spot on this! I believe this is the first occurance of CTAD in the node codebase. Double checked that this is confirming to our style guide: https://github.com/nodejs/node/pull/56071.

Reply:
Templates in `std` are allowed as they explicitly defined deduction guides. This is good.